charts/spinnaker/values.yaml

```diff
@@ -170,43 +170,36 @@ kubeConfig:
   # Use this when you want to register arbitrary clusters with Spinnaker
   # Upload your ~/kube/.config to a secret
   enabled: false
-  # Creates a Spinnaker account per context in `.kubeConfig.contexts` with the
-  # same name. Can be used together with
-  # `.kubeConfig.accountPerContextAndNamespace`.
-  accountPerContext: true
-  # Creates a Spinnaker account for each combination of context and namespace
-  # as per `contexts` and `namespacePartitionedAccounts`. Can be used together
-  # with`accountPerContext`.
-  accountPerContextAndNamespace: false
   # secretName: my-kubeconfig
   # secretKey: config
   # Use this when you want to configure halyard to reference a kubeconfig from s3
   # This allows you to keep your kubeconfig in an encrypted s3 bucket
   # For more info visit:
   #   https://www.spinnaker.io/reference/halyard/secrets/s3-secrets/#secrets-in-s3
   # encryptedKubeconfig: encrypted:s3!r:us-west-2!b:mybucket!f:mykubeconfig
-  # List of contexts from the kubeconfig to make available to Spinnaker
-  contexts:
-  - default
+  accounts:
+    # default:
```

Comment:
I have to comment this out (as opposed to before) as we iterate over this map, and append to it in our own deployment files. Otherwise, we'd create a `default` account.

Reply:
Alternatively we could set it to `null` in our internal deployment files before applying the layering. But I think leaving it like this is also OK.